java/opendcs/src/main/java/decodes/tsdb/TimeSeriesDb.java

```diff
@@ -193,6 +194,16 @@ public TimeSeriesDb(String appName, javax.sql.DataSource dataSource, DecodesSett
         {
             determineTsdbVersion(conn, this);
             postConnectInit(appName, conn);
+            
+			DatabaseMetaData meta = conn.getMetaData();
+			if (meta.getDatabaseProductName().contains("PostgreSQL"))
```

Comment:
Consider if a DRY approach for similar code in SqlDatabaseIO constructor would work.

Reply:
It will eventually. Until we've removed all of the SqlDatabaseIO based classes it's a fair amount of work to avoid some of that duplication.